### PR TITLE
DVX-6275 Export Embedded elements for Facebook Instant Articles

### PR DESCRIPTION
**Pull request by towanda**
Export embedded elements (tweets, youtube videos, etc) for Instant Articles
following the reference document:
https://developers.facebook.com/docs/instant-articles/reference/embeds

### CODE REVIEW

**Comment by miguelbernadi** [comment]
+1

**Comment by christofdamian** [comment]
+1

File: lib/article_json/export/facebook_instant_article/elements/embed.rb [hunk 1]
```diff
@@ -3,7 +3,40 @@ module Export
...
+            end
+          end
+
+          private
+
+          # Type specific object that should be embedded
+          # @return [Nokogiri::XML::Element]
+          def embed_node
+            if %i(facebook_video tweet).include? @element.embed_type.to_sym
+              default_node
```
Comment (dsager) [inline lib/article_json/export/facebook_instant_article/elements/embed.rb]: I would expect the default to be in the `else` branch, and not for two specific cases...

File: lib/article_json/export/facebook_instant_article/elements/embed.rb [hunk 2]
```diff
@@ -3,7 +3,40 @@ module Export
...
+            else
+              embedded_object
+            end
+          end
+
+          def default_node
```
Comment (dsager) [inline lib/article_json/export/facebook_instant_article/elements/embed.rb]: what about naming this `iframe_node`?

**Review by dsager — APPROVED** [review]

File: lib/article_json/export/facebook_instant_article/elements/embed.rb (hunk 1 above)
Comment (towanda) [reply lib/article_json/export/facebook_instant_article/elements/embed.rb]: I changed the method name to 'iframe_node'.

File: lib/article_json/export/facebook_instant_article/elements/embed.rb (hunk 2 above)
Comment (towanda) [reply lib/article_json/export/facebook_instant_article/elements/embed.rb]: good point, changed

File: lib/article_json/export/facebook_instant_article/elements/embed.rb (hunk 1 above)
Comment (dsager) [reply lib/article_json/export/facebook_instant_article/elements/embed.rb]: thanks!

**Pull request merged**